hooks/useServicesListsHandler.ts

```diff
@@ -0,0 +1,45 @@
+import { useMemo } from "react"
+import { useQuery } from "react-query"
+import { Column } from "react-table"
+import { getServicesLists } from "../api"
+import { ServicesList } from "../models"
+
+export interface ServicesListsHandler {
+  servicesListsData: object[]
+  servicesListsColumns: Column[]
+  isLoading: boolean
+}
+
+export const useServicesListsHandler = (): ServicesListsHandler => {
+  const { 
+    isLoading,
+    isError,
+    data,
+  } = useQuery<ServicesList[], Error>("servicesLists", () => getServicesLists("Status='Published'"))
+
+  const servicesListsColumns = useMemo(
+    () => [
+      {
+        Header: "Name",
+        accessor: "Name"
+      },
+      {
+        Header: "Services",
+        accessor: "Services"
+      },
+      {
+        Header: "Owner",
+        accessor: "Owner"
+      },
+    ], 
+    []
+  )
+
+  const servicesListsData = useMemo(() => data, [data])
```

Comment:
The data passed into the React Table's `useTable()` hook must be memoized according to the [React Table docs](https://react-table.tanstack.com/docs/quick-start#using-the-usetable-hook) - maybe it's clearer if we move `useMemo()` call further down?